docs/index.rst

```diff
@@ -37,4 +37,5 @@ Acknowledgements
 .. _`Lauren Ko`: http://github.com/ldko
 .. _`Mark Phillips`: http://github.com/vphill
 .. _`Gio Gottardi`: http://github.com/somexpert
+.. _`Madhulika Bayyavarapu`: http://github.com/madhulika95b
```

Comment:
May as well update all of these to https, yeah?

Reply:
ok. Doing that wouldn't hurt. I'll update all of them.